src/packages/@ncigdc/modern_components/Query.js

```diff
@@ -14,7 +14,7 @@ export const BaseQuery = (props: Object) =>
 
       lastRelayProps[props.name] = nextRelayProps || lastRelayProps[props.name];
 
-      if (lastRelayProps[props.name])
+      if (lastRelayProps[props.name] || (props.renderLoading && !error))
```

Comment:
@alex-wilmer not sure if this makes sense. 
I realized that it still wasn't right aligned in the loading state because it was a different component. 
Probably a lot of components can handle their own loading state, so this is my first stab at it.